client/src/tokens/objectStyles.js

```diff
@@ -10,7 +10,8 @@ const borderRadius = {
 const borderWidth = {
   DEFAULT: '0.0625rem', // 1px
   0: '0',
-  5: '0.3125rem',
+  2: '0.125rem', // 2px
```

Comment:
We have quite a few 2px borders already but apparently only in Sass where we don’t check the variety of border widths.